oat-sa/tao-core
# feat: Add form data provider to speed up forms

Ticket - https://oat-sa.atlassian.net/browse/REL-1323

Related PR - https://github.com/oat-sa/generis/pull/1090

How to test:
* Checkout to a new code
* Don't forget to rebuild a container if necessary `php index.php 'oat\generis\scripts\tools\ContainerCacheWarmup'`
* Create a nested item or test form with a lot of properties
* The form should be loaded faster! (or check the number or database queries during form loading request)

In order to switch to the previous (slow) implementation these lines can be commented:
https://github.com/oat-sa/tao-core/pull/3929/files?diff=unified&w=0#diff-c0cbe776170fe513f091b0c7cc304b4448c11c8db33c52d1501043981904aca9R45-R47
